examples/index_searchball.html

```diff
@@ -0,0 +1,192 @@
+
+					var id = geometry.attributes.id.array[ tile ];
+					var i = Math.floor( id / size );
+					var u0 = id / size - i;
+					var v0 = 1. - ( i + 1. ) / size;
+
+					geometry.attributes.uv.array[ 2 * tile ] *= ( 1 - 2 * cut ) / size;
+					geometry.attributes.uv.array[ 2 * tile ] += u0 + cut / size;
+					geometry.attributes.uv.array[ 2 * tile + 1 ] *= ( 1 - 2 * cut ) / size;
+					geometry.attributes.uv.array[ 2 * tile + 1 ] += v0 + cut / size;
+
+				}
+
+				var material = new THREE.MeshBasicMaterial( { color: 0xAAAAAA } );
+				var mesh = new THREE.Mesh( geometry, material );
+				new THREE.TextureLoader().load( './screenshots/all_in_one.jpg', function( tex ) {
+					material.color = null;
+					material.map = tex;
+					material.needsUpdate = true;
+				});
+
+				scene = new THREE.Scene();
+				scene.add( mesh );
+				scene.background = new THREE.Color( 0xFFFFFF );
+
+				var directionalLight = new THREE.DirectionalLight( 0xFFFFFF, 1 );
+				directionalLight.position.set( 1, 1, 1 ).normalize();
+				scene.add( directionalLight );
+
+				// renderer
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				document.body.appendChild( renderer.domElement );
+
+				// controls
+
+				controls = new OrbitControls( camera, renderer.domElement );
+
+				controls.enableDamping = true;
+				controls.dampingFactor = 0.04;
+
+				controls.screenSpacePanning = false;
+				controls.enablePan = false;
+
+				controls.minDistance = 1.05;
+				controls.maxDistance = 10;
+
+				// resize
+
+				if ( camera.aspect < 1 ) {
+
+					var pos = camera.position.divideScalar( camera.aspect );
+					camera.position.set( pos.x, pos.y, pos.z );
+
+				}
+				window.addEventListener( 'resize', onWindowResize, false );
+				window.addEventListener( 'mousemove', onMouseMove, false );
+				window.addEventListener( 'click', onMouseClick, false );
+
+			}
+
+			function quickRaycaster( camera, mouse, radius, turns, tiles ) {
+
+				var ray = new THREE.Ray();
+				ray.origin.setFromMatrixPosition( camera.matrixWorld );
+				ray.direction.set( mouse.x, mouse.y, 0.5 ).unproject( camera ).sub( ray.origin ).normalize();
+
+				var a = ray.direction.dot( ray.direction );
+				var b = 2 * ray.origin.dot( ray.direction );
+				var c = ray.origin.dot( ray.origin ) - radius * radius;
+				var D = b * b - 4 * a * c;
+
+				if ( D < 0 ) {
+
+					return null;
+
+				}
+
+				var p = new THREE.Vector3();
+				var s = ( - b - Math.sqrt( D ) ) / ( 2 * a );
+				ray.at( s, p );
+				p = new THREE.Spherical().setFromVector3( p );
+
+				p.phi = Math.PI / 2 - p.phi;
+				var t = p.theta / 2 / turns + Math.PI / turns *
+						Math.floor( ( turns * ( Math.PI - 2 * p.phi ) - p.theta ) / ( 2 * Math.PI ) ) - Math.PI / 2;
+				var id = 1 + Math.floor( ( tiles - 1 ) * ( Math.cos( Math.PI / 2 / turns ) -
```

Comment:
@looeee We can trace a sphere, find spherical coordinates of a point and than take a formula in paper by Marco Abrate and Fabrizio Pollastri.